### ISSUE TITLE
Create Button Bug

### ISSUE THREAD

**Issue by daveyshindig** [orionjs/orion#350]
First of all, thanks for building this project for the Meteor community. I'm really excited to get it working with my app, and would like to contribute to its development if I can. But I am having some trouble and suspect there is some kind of bug...

When creating a post through the Admin panel, the "Create" button doesn't submit the form. For an example, please checkout the microphone-orion tutorial: https://github.com/fuzzybabybunny/microscope-orionjs

One thing is really strange: in that project, it's possible to create a new "Post," but not a new "Comment." Saving edits to preexisting Comments and Posts works fine, too.  When you create a new post, clicking Create validates the form and gives errors if they're missing or formatted incorrectly, but other than that it doesn't seem to do anything. 

I don't understand why this is the case. The console shows no errors, and none of the debugging I've tried has turned up an answer. I've tried a few different browsers and they all have the same problem. Is this a bug?

**Comment by lusilva** [comment]
I'm also experiencing this issue

**Comment by nicolaslopezj** [comment]
Did this problem disappear?

**Comment by daveyshindig** [comment]
Yes, at least in my later efforts it doesn't seem to be a problem.
